.changeset/young-ants-nail.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+'@backstage/catalog-model': patch
```

Comment:
Don't think we need this here anymore as we're not touching this package anymore 👍

Reply:
good call 👍